index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Hello Microverse Project</title>
+  </head>
+  <body>
+    <h1>Hello Microverse!</h1>
```

Comment:
- [ ] It is required to give your `h1` element a class name and use that class name to style the content of the `h1`. Kindly add a class name to your element and use that class name as a reference to style the content of your index file from css file.